### PR TITLE
feature, python: allow for users to define custom exports from __init__.py

### PR DESCRIPTION
**Pull request by armandobelardo**
This also adds docstrings to root clients for usability
And adds in flags for `fern write-overrides` to be initialize model names, as well as point to an existing overrides file to start from

### CODE REVIEW

File: packages/cli/cli/src/cli.ts [hunk 1]
```diff
@@ -720,20 +720,32 @@ function addWriteOverridesCommand(cli: Argv<GlobalCliOptions>, cliContext: CliCo
...
             yargs.option("api", {
                 string: true,
                 description: "Only run the command on the provided API"
             }),
+            yargs.option("include-models", {
+                boolean: true,
+                description:
+                    "When generating the initial overrides, also stub the models (in addition to the endpoints)"
+            }),
+            yargs.option("existing-overrides", {
```
Comment (dsinghvi) [inline packages/cli/cli/src/cli.ts:733]: can we detect this by looking at their `generators.yml` ? You should be able to get access to that from the TaskContext

File: packages/cli/cli/src/cli.ts [hunk 2]
```diff
@@ -720,20 +720,32 @@ function addWriteOverridesCommand(cli: Argv<GlobalCliOptions>, cliContext: CliCo
     cli.command(
         "write-overrides",
         "Generate a basic openapi overrides file.",
-        (yargs) =>
+        (yargs) => [
             yargs.option("api", {
                 string: true,
                 description: "Only run the command on the provided API"
             }),
+            yargs.option("include-models", {
```
Comment (dsinghvi) [inline packages/cli/cli/src/cli.ts:728]: Let's just always default this to true for now

Comment (dsinghvi) [reply packages/cli/cli/src/cli.ts:728]: Can always delete what we don't like

File: generators/python/sdk/CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,45 @@ All notable changes to this project will be documented in this file.
...
+    """
+    Use this class to access the different functions within the SDK. You can instantiate any number of clients with different configuration that will propogate to these functions.
+    ---
+    from seed.client import SeedTest
+
+    client = SeedTest(
+        token="YOUR_TOKEN",
+        base_url="https://yourhost.com/path/to/api",
+    )
+    """
```
Comment (dsinghvi) [inline generators/python/sdk/CHANGELOG.md:44]: do you think it's easy to add the parameter level documentation as well just like we do for endpoints? 

```python
token: Optional[str]. The token. 

base_url: Optional[str]. The Sever URL used by the SDK. 
```

**Review by dsinghvi — CHANGES_REQUESTED** [review]
Couple small comments. Let's all add a snapshot test in `cli/ete-tests` that calls `write-overrides` !

File: generators/python/sdk/CHANGELOG.md (hunk 3 above)
Comment (armandobelardo) [reply generators/python/sdk/CHANGELOG.md:44]: Let me check!

File: packages/cli/cli/src/cli.ts (hunk 2 above)
Comment (armandobelardo) [reply packages/cli/cli/src/cli.ts:728]: totally

File: generators/python/sdk/CHANGELOG.md (hunk 3 above)
Comment (armandobelardo) [reply generators/python/sdk/CHANGELOG.md:44]: Updated!

**Review by dsinghvi — APPROVED** [review]

**Pull request merged**